src/components/ActNowSummary/index.js

```diff
@@ -0,0 +1,56 @@
+import { Grid, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import PropTypes from "prop-types";
+import React from "react";
+
+const useStyles = makeStyles(() => ({
+  root: {
+    display: "flex",
+    backgroundColor: "#F7F7F7",
+    padding: "1rem",
+    justifyContent: "space-around",
+    alignItems: "flex-end",
+  },
+  count: {
+    fontSize: "1.2rem",
+    fontWeight: 700,
+  },
+}));
+
+function ActNowSummary({ className, summary, ...props }) {
```

Comment:
Are we using className in this component?

Reply:
No. I'll remove it